src/Connection.js

```diff
@@ -4,16 +4,15 @@ import mysql from 'mysql';
 
 export default class Connection {
 
-    constructor (handle) {
+    constructor (handle, key) {
```

Comment:
When do we pass-in the key? (Also I don't see a test code for this?)

Reply:
in case we want to specify the key to use and not the default

Reply:
if key is missing, it uses the default

Reply:
Yes, I understand what the code does, I'm asking for the specific use case scenario and test cases. 
The user should never really need to create a connection of their own, right?